activerecord/test/cases/relation/load_async_test.rb

```diff
@@ -231,12 +231,10 @@ class LoadAsyncNullExecutorTest < ActiveRecord::TestCase
       def setup
         @old_config = ActiveRecord.async_query_executor
         ActiveRecord.async_query_executor = nil
-        ActiveRecord::Base.establish_connection :arunit
```

Comment:
I removed these because I found that the sqlite3 async tests were failing with some seeds and my changes. After debugging I found that the connection that was opened had `asycn_query_executor` set to `nil` rather than the global executor default. I don't know why these tests were re-establishing a connection, there would be one in the pool to pick up. Removing this fixed the tests that incorrectly had the wrong query executor. I believe this works because it will open a new connection for both tests (there wont be one in the pool). Previously calling `establish_connection` was ensuring there was an existing pool (with async executor set to nil) but we don't want to use that one so it's better not to establish a connection at all.